<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="btnProcesar">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Procesar!</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTexto">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblResultado">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>150</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Resultado:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTexto">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Texto:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtResultado">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="lstTexto">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkAddLista">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar a la Lista</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar Lista</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIniciar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
